CMakeLists.txt

```diff
@@ -31,9 +31,9 @@
 ############################################################
 
 cmake_minimum_required(VERSION 3.3)
-set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "${CMAKE_SOURCE_DIR}/cmake/Modules/")
+set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "${CMAKE_CURRENT_SOURCE_DIR}/cmake/Modules/")
 
-set(TILEDB_CMAKE_INPUTS_DIR "${CMAKE_SOURCE_DIR}/cmake/inputs")
+set(TILEDB_CMAKE_INPUTS_DIR "${CMAKE_CURRENT_SOURCE_DIR}/cmake/inputs")
 
```

Comment:
It might be nice here to define a `TILEDB_ROOT_SOURCE_DIR` variable which can then be used in subdirectories instead of having to do `${CMAKE_CURRENT_SOURCE_DIR}/..` in several places.

Reply:
After some discussion with @jakebolewski it's fine to leave this as-is (don't worry about adding the new variable). He also made the point that in navigating nested CMake projects, the relative path may actually be clearer (rather than having to look up the definition of a project-specific variable).

Reply:
I've left this as `CMAKE_CURRENT_SOURCE_DIR` and not moved to `TILEDB_ROOT_SOURCE_DIR`. I agree with @jakebolewski that using `CMAKE_CURRENT_SOURCE_DIR` makes it easy to know the path based on the `CMakeLists.txt` file that is being examined.